src/pages/NIHicWebform.js

```diff
@@ -0,0 +1,2266 @@
+                        })
+                      ])
+                  ]),
+
+                  div({className: 'form-group'}, [
+                    div(
+                      {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'},
+                      [
+                        label({className: 'control-label rp-title-question common-color'}, [
+                          'Do you have an eRA Commons Account?',
+                        ]),
+                      ]),
+                    div({ className: 'row no-margin' }, [
+                      div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 dataset-group' }, [
+
+                        RadioButton({
+                          style: {
+                            margin: '2rem',
+                            color: ' #010101',
+                          },
+                          id: 'eraAuthorizedYes',
+                          name: 'eraAuthorizedYes',
+                          value: 'yes',
+                          defaultChecked: true,
+                          onClick: () => {},
+                          label: 'Yes',
+                          disabled: false,
+                        }),
+
+                        RadioButton({
+                          style: {
+                            marginBottom: '2rem',
+                            marginLeft: '2rem',
+                            color: ' #010101',
+                          },
+                          id: 'eraAuthorizedNo',
+                          name: 'eraAuthorizedNo',
+                          value: 'no',
+                          defaultChecked: false,
+                          onClick: () => {},
+                          label: 'No',
+                          disabled: false,
+                        }),
+                      ]),
+                    ]),
+                  ]),
+
+                  div({className: 'form-group'}, [
+                    div(
+                      {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'},
+                      [
+                        label({className: 'control-label rp-title-question common-color'}, [
+                          'NIH Grant or Contract Number ',
+                        ]),
+                      ]),
+                    div(
+                      {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group rp-last-group'},
+                      [
+                        input({
+                          type: 'text',
+                          name: 'description',
+                          id: 'inputDescription',
+                          maxLength: '256',
+                          value: this.state.datasetData.description,
+                          onChange: () => {},
+                          className: (fp.isEmpty(this.state.datasetData.description) && showValidationMessages) ?
+                            'form-control required-field-error' :
+                            'form-control',
```

Comment:
This should just be: 
```suggestion
                          className: 'form-control',
```